### PR TITLE
Two non-sober hereditarily connected spaces

### PR DESCRIPTION
**Pull request by prabau**
Related to some of the discussion in #934.

Adding two closely related spaces to satisfy the query:
[π-Base, Search for `hereditarily connected + generic point + focal point + ~sober + t0`](https://topology.pi-base.org/spaces?q=hereditarily+connected+%2B+generic+point+%2B+focal+point+%2B+%7Esober+%2B+t0)

- S150: Right "closed-ray" topology on $[0,1]\cap\mathbb Q$
- S151: Right "open-ray" topology on $[0,1]\cap\mathbb Q$

Not sure if the names could cause confusion, but that's what I came up to distinguish them.  We can discuss alternatives.

I did not add aliases for "left ray topologies", but they would be homeomorphic in this case.

### CODE REVIEW

**Comment by prabau** [comment]
One missing trait is for the Baire property.

That should follow from a theorem [has generic point ==> Baire], which I'll add in a separate PR.

**Comment by prabau** [comment]
@StevenClontz Since you previously added a few ray topologies, maybe you'll have an opinion about the names.

**Comment by GeoffreySangston** [comment]
I looked through this yesterday and saw nothing to comment on. I'll look through it again now and then submit the review.

File: spaces/S000150/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+uid: S000150
+name: Right "closed-ray" topology on $[0,1]\cap\mathbb Q$
+---
+
+The set $X=[0,1]\cap\mathbb Q$ with its order induced from $\mathbb R$, and the topology generated by a base of open sets consisting of the intervals $[x,\rightarrow)$ for $x\in X$.
+
+The open sets are exactly the [upward-closed sets](https://en.wikipedia.org/wiki/Upper_set) in the ordered set $(X,\le)$.  That includes "closed rays" $[x,\rightarrow)$ and "open rays" $(x,\rightarrow)$ for $x\in X$, as well as sets of the form $(\theta,1]\cap\mathbb Q$ for an irrational $\theta\in(0,1)$.
```
Comment (GeoffreySangston) [inline spaces/S000150/README.md:8]: Should we add the parallel section to 151?

**Review by GeoffreySangston — APPROVED** [review]
I did not see any mistakes.

File: spaces/S000150/README.md (hunk 1 above)
Comment (prabau) [reply spaces/S000150/README.md:8]: Yeah, I thought about it.  It's kind of obvious that sets of the form $(x,\rightarrow)$ and  $(\theta,,1]\cap X$ are open.
But maybe we could mention:
"Note that "closed rays" of the form $[x,\rightarrow)$ for $x\in X\setminus\\{0\\}$ are not open sets."

Thoughts?

**Comment by prabau** [comment]
@StevenClontz Since you previously added a few ray topologies, maybe you'll have an opinion about the names.

**Comment by prabau** [comment]
> I did not see any mistakes.

The purported base for the topology was not a base.  Was missing $X$, now fixed.

**Pull request merged**